File: dataset/avatar/Python/Code/codeforces_450_A.py
n , m = map ( int , input ( ).split ( ) )
l = list ( map ( int , input ( ).split ( ) ) )
l2 = [ ]
for i in l :
    if i % m == 0 :
        l2.append ( i // m )
    else :
        l2.append ( i // m + 1 )
mx = max ( l2 )
for i in range ( n ) :
    if l2 [ i ] == mx :
        ind = i
print ( ind + 1 )

File: output/deepseek_r1/avatar/direct_translation/python/java/codeforces_450_A.java

import java.util.Scanner;

public class codeforces_450_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        sc.nextLine();
        String[] parts = sc.nextLine().split(" ");
        int[] l = new int[n];
        for (int i = 0; i < n; i++) {
            l[i] = Integer.parseInt(parts[i]);
        }
        int[] l2 = new int[n];
        for (int i = 0; i < n; i++) {
            if (l[i] % m == 0) {
                l2[i] = l[i] / m;
            } else {
                l2[i] = l[i] / m + 1;
            }
        }
        int mx = l2[0];
        for (int num : l2) {
            if (num > mx) {
                mx = num;
            }
        }
        int ind = 0;
        for (int i = 0; i < n; i++) {
            if (l2[i] == mx) {
                ind = i;
            }
        }
        System.out.println(ind + 1);
    }
}

